pop-api/src/v0/contracts.rs

```diff
@@ -0,0 +1,156 @@
+use crate::{
+    PopApiError,
+    PopApiError::UnknownStatusCode,
+};
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, scale::Encode, scale::Decode)]
+#[cfg_attr(feature = "std", derive(scale_info::TypeInfo))]
+pub enum Error {
```

Comment:
I don't get why we have these pallet-contracts errors in the Pop API? 

These errors will occur when the user calls the contract. Most of this will not apply to the contract calling things.

Reply:
Please don't review, sorry it probably still was on review from before. 

To give some context, there will be errors that can happen by calling a fungible api which is not from pallet assets. Currently researching this.